extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/MapboxCarMapSessionInstaller.kt

```diff
@@ -93,7 +96,21 @@ class MapboxCarMapSessionInstaller(
     return mapboxCarMap
   }
 
+  private fun enforceSharedContext(mapInitOptions: MapInitOptions) {
+    if (mapInitOptions.mapOptions.contextMode != ContextMode.SHARED) {
+      logW(
+        TAG,
+        "Explicitly switching to MapOptions.contextMode = ContextMode.SHARED when creating the car map surface. " +
+          "ContextMode.UNIQUE (used by default) is not allowed as it leads to graphical artifacts and crashes."
```

Comment:
```suggestion
        "Explicitly switching to MapOptions.contextMode = ContextMode.SHARED. " +
          "ContextMode.UNIQUE applies optimizations that do not work with BitmapWidget."
```

nitpick on the warning. considering that BitmapWidget is still experimental and this issue could still be addressed, I think it's worth calling it out specifically

Reply:
Also wondering if it should just be handled inside and not surface the warning. This issue seems like it'll be temporary, and then we will need to tell developers to remove configuration after a future release.

Reply:
Also I don't like how this warning is sent with the default configuration. I think my opinion is to remove the warning entirely, and use a comment instead.

Reply:
Removed the warning when no context mode is specified explicitly by the user (assume it's 99.9% of use-cases) but left it when user is setting unique - then we force using shared + print a warning about it; also updated unit tests